examples/python/util.py

```diff
@@ -0,0 +1,28 @@
+import numpy as np
+import scipy.sparse as spa
+
+
+def generate_mixed_qp(n, seed=1, reg=-2.0, dens1=0.075):
```

Comment:
`reg` is not used. we should either remove it, as it is not used by any of our examples for the dense backend, or give it a meaning like [here](https://github.com/Simple-Robotics/proxsuite/blob/main/test/src/sparse_qp_wrapper.py#L34) for future usage.

Reply:
You're right, it was already in the code but never used.

Reply:
done